### ISSUE TITLE
Sort icons are missing

### ISSUE THREAD

**Issue by joebordes** [metabase/metabase#6229]
- Your browser and the version: Fails both on Firefox and Chrome with the same error
- Your operating system: Linux Mint Sarah
- Your databases: H2 Sample Data
- Metabase version: 0.26.2
- Metabase hosting environment: Linux
- Metabase internal database: H2 (default)

- Sort by any column

-------

I just upgraded to 0.26.2 and I lost the sort icons in the header columns of questions.

![missingsorticon](https://user-images.githubusercontent.com/1237191/31861688-d7534f86-b731-11e7-875f-2c91e51f37e6.png)

I see this error in the javascript console:

```
webfont.js:31 Refused to load the font 
'data:font/woff;base64,d09GRgABAAAAAIw4ABEAAAABQcAAAQABAAAAAAAAAAAAAAAAAAAAA…hgZiCKVViwAiVhsAFFYyNisAIjRLMJCgMCK7MLEAMCK7MRFgMCK1myBCgGRVJEswsQBAIrAA=='
 because it violates the following Content Security Policy directive: "font-src 'self' fonts.gstatic.com themes.googleusercontent.com ".
```

**Comment by mazameli** [comment]
Hi, Joe. If you click on the heading, like `Address`, you should see an action menu which includes sort options. Is that working for you?

**Comment by joebordes** [comment]
Yes, the menu is there and it actually sorts correctly but there is no chevron character or "down"/"up" arrow to indicate that the column has been sorted

**Comment by mazameli** [comment]
Ohhh, I see. I misunderstood what the issue was that you were reporting.

**Comment by salsakran** [comment]
fixed via #6740
